docs/changehistory/NextVersion.md

```diff
@@ -13,9 +13,11 @@ Table of contents:
   - [Renderer, editor and category on calculated properties](#renderer-editor-and-category-on-calculated-properties)
 - [Electron 25 support](#electron-25-support)
 - [Geometry](#geometry)
-  - [Sweeping a section to a sequence of planes](#sweeping-a-section-to-a-sequence-of-planes)
+  - [Sweeping a section to a sequence of planes]
+  - [Sweeping a linestring to facets]
```

Comment:
```suggestion
  - [Sweeping a section to a sequence of planes](#sweeping-a-section-to-a-sequence-of-planes)
  - [Sweeping a linestring to facets](#sweeping-a-linestring-to-facets)
```

Reply:
These are supposed to be links. The first one was a link prior to your edit.

Reply:
Unless you object to proposed changes, you can just use the "Commit suggestion" buttons to apply them.